src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/Helper.WinRT.cs

```diff
@@ -2,7 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
```

Comment:
@JeremyKuhne, are we not going to use the same binary across all Windows?  I thought that was a (new) goal.